addons/languages/urdu/apk/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.anysoftkeyboard.languagepack.urdu">
```

Comment:
do you want to change the `package` to `com.anysoftkeyboard.languagepack.mirfatif.urdu` in the `apk` module? I think it will make things less confusing

Reply:
the `pack` module can stay as is, it doesn't matter